Add information on Social Preview Image (#291)
nf-core/nf-co.re: markdown/developers/adding_pipelines.md

in the nf-core [Repository health web page](https://nf-co.re/pipeline_health).
This reports the status of various checks and also has the option of fixing errors for you via the GitHub API.

### Social Preview Image

From the [nf-core/logos repository](https://github.com/nf-core/logos), download the [social preview image template](https://github.com/nf-core/logos/blob/master/nf-core-logos/social_preview_image_template.svg), and follow these instructions:

* Open the `social_preview_image_template.svg` file using [Inkscape](https://inkscape.org/) or any other vector graphics editor
* You might need to install the google font [Maven Pro](https://fonts.google.com/specimen/Maven+Pro)
* Update the pipeline name

> * If, and only if pipeline name is too long then center the logo

Reviewer: ```suggestion
* If the pipeline name goes beyond the apple, centre the logo in the image
```

* Update the pipeline description
* Save as `assets/nf-core-<pipeline-name>_social_preview.svg`
* Export as `assets/nf-core-<pipeline-name>_social_preview.png`
* Upload it on the nf-core pipeline GitHub repository in the _Social Preview_ section in the _Settings_ tab

Unfortunately, these steps cannot be automated, as the description of the pipeline could take one or two lines.

> Line break should not 

Reviewer: ..?

Author: I must not have saved the right version of this phrase...

> Line break should not 

Author: ```suggestion
Line break should look nice and be readable
```

> Line break should look nice and be readable

Author: ```suggestion
Line break should look nice and be readable.
```
